source/layouts/api_catalogue.erb

```diff
@@ -0,0 +1,7 @@
+<% wrap_layout(:layout) do
+  content_for(:head) do
+    '<script async defer data-domain="alphagov.github.io/api-catalogue" src="https://plausible.io/js/plausible.js"></script>'
```

Comment:
@charlesbaird-dsa I notice the 'data domain' attribute in the script you provided includes a path (`/api-catalogue`). Did you grab this snipped from the Plausible site?

There's a chance this should just be `alphagov.github.io` because the script just checks the host name. But guess we'll find out when it's deployed...

Reply:
Yes - that came from the Plausible site. I guess we'll have to see when it's running...

Looks good, thanks.